src/main.rs

```diff
@@ -108,10 +112,7 @@ fn main() {
                     let path = it["path"].as_str().unwrap();
                     let value = if path.ends_with('/') || path.ends_with("\\"){ pop(path) }else{ path };
                     event_parent_path.contains(value) &&
-                    !event_filename.to_str().unwrap().contains(match it["ignore"].as_str(){
-                        Some(ig) => ig,
-                        None => "?"
-                    })
+                    !it["ignore"].as_vec().unwrap().iter().any(|ignore| { event_filename.to_str().unwrap().contains(ignore.as_str().unwrap()) })
```

Comment:
I have tested the code. When the ignore is empty it panics, add this to avoid it 😃 
```suggestion
                    !match it["ignore"].as_vec(){
                        Some(ig) => ig.iter().any(|ignore| { event_filename.to_str().unwrap().contains(ignore.as_str().unwrap()) }),
                        None => true
                    }
```